domains/pherngpy.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "shockbs",
+        "email": "apipherng@gmail.com"
+    },
+    "record": {
+        "CNAME": "willchangelater"
```

Comment:
placeholders are not allowed

Reply:
sorry my project aren't ready, i will keep changing servers for this domain so i want to ask my two domains are approved, where should i configure it? it's not showing at manage.is-a.dev

> placeholders are not allowed

Reply:
> placeholders are not allowed

done